tests/40-jcs-create.js

```diff
@@ -0,0 +1,178 @@
+              verificationMethodDocument?.type === 'Multikey'
+          ).should.equal(true, 'Expected at least one proof to have "type" ' +
+            'property value "Multikey".');
+        });
+        it('The "proof.proofPurpose" field MUST match the verification ' +
+          'relationship expressed by the verification method controller.',
+        async function() {
+          this.test.cell = {columnId, rowId: this.test.title};
+          verificationMethodDocuments.should.not.eql([], 'Expected ' +
+            'at least one "verificationMethodDocument".');
+          verificationMethodDocuments.some(
+            verificationMethodDocument =>
+              verificationMethodDocument?.type === 'Multikey'
+          ).should.equal(true, 'Expected at least one proof to have "type" ' +
+            'property value "Multikey".'
+          );
+          const controllerDocuments = [];
+          for(const verificationMethodDocument of verificationMethodDocuments) {
+            const controllerDocument = await documentLoader({
+              url: verificationMethodDocument.controller
+            });
+            controllerDocuments.push(controllerDocument);
+          }
+          proofs.some(
+            proof => controllerDocuments.some(controllerDocument =>
+              controllerDocument.hasOwnProperty(proof.proofPurpose))
+          ).should.equal(true, 'Expected "proof.proofPurpose" field ' +
+             'to match the verification method controller.'
+          );
+        });
+        it('The "publicKeyMultibase" value of the verification method MUST ' +
+          'be 34 bytes in length and starts with the base-58-btc prefix (z).',
+        async function() {
+          this.test.cell = {columnId, rowId: this.test.title};
+          verificationMethodDocuments.should.not.eql([], 'Expected ' +
+            'at least one "verificationMethodDocument".');
+          for(const verificationMethodDocument of verificationMethodDocuments) {
+            const multibase = 'z';
+            const {publicKeyMultibase} = verificationMethodDocument;
+            const isMultibaseEncoded =
+              publicKeyMultibase.startsWith(multibase) &&
+                shouldBeBs58(publicKeyMultibase);
+            isMultibaseEncoded.should.equal(
+              true,
+              'Expected "publicKeyMultibase" value of the verification ' +
+              'method to be multibase base58-btc encoded value'
+            );
+            const publicKeyMultibaseBytes = bs58Decode({
+              id: publicKeyMultibase
+            });
+            publicKeyMultibaseBytes.byteLength.should.equal(34, 'Expected ' +
+              '"publicKeyMultibase" value of the verification method to ' +
+              'be 34 bytes in length.');
+          }
+        });
+        it('"proofValue" field when decoded to raw bytes, MUST be 64 bytes ' +
+          'in length if the associated public key is 32 bytes or 114 bytes ' +
```

Comment:
```suggestion
          'in length if the associated public key is 32 bytes, or 114 bytes ' +
```

Reply:
This is a verbatim text from the specs and the test description will change in future, so leaving this as it is for now.